<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Allportdata</class>
 <widget class="QWidget" name="Main_Allportdata">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
    font:12px;
}
QLabel
{
color:white;
}QPushButton
{
font: 16px;color: rgb(192,192,192);background-color: rgb(0, 0, 0);border-top: 2px solid white;border-left: 2px solid white;
}</string>
  </property>
  <widget class="QPushButton" name="BTNDDU">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>30</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>DDU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNTCN1">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>80</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TCN1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNTCN2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>130</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TCN2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNTCN4">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>230</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TCN4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNTCN3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>180</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TCN3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNTCU">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>330</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TCU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNRIOM">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>280</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RIOM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNOTHER">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>380</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OTHER</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
